packages/devtools/lib/src/vm_service_wrapper.dart

```diff
@@ -23,8 +23,9 @@ class VmServiceWrapper implements VmService {
   final Map<String, Future<Success>> _activeStreams = {};
 
```

Comment:
add a top level bool field to this file like
String debugFutures;
that toggles on and off whether we track the Map<Future, String> futureCreators;

Reply:
It would probably make sense to set
debugFutures=true
for all our tests.
Outside of tests, this functionality isn't really useful like you say.

Reply:
Done!